.ci/build-mac.sh

```diff
@@ -40,7 +43,15 @@ export CPPFLAGS="-I$BREW_X64_PATH/include -msse -msse2 -mcx16 -no-pie"
 export LIBRARY_PATH="$BREW_X64_PATH/lib"
 export LD_LIBRARY_PATH="$BREW_X64_PATH/lib"
 
-git submodule update --init --recursive --depth 1
+export VULKAN_SDK
+VULKAN_SDK="$BREW_X64_PATH/opt/molten-vk"
+ln -s "$VULKAN_SDK/lib/libMoltenVK.dylib" "$VULKAN_SDK/lib/libvulkan.dylib"
+export VK_ICD_FILENAMES="$VULKAN_SDK/share/vulkan/icd.d/MoltenVK_icd.json"
+
+export LLVM_DIR
+LLVM_DIR="BREW_X64_PATH/opt/llvm@16"
+# exclude FAudio, SPIRV and LLVM from submodule update
+git submodule foreach 'if [[ $path != "3rdparty/FAudio" && $path != "3rdparty/llvm" && $path != "3rdparty/SPIRV" ]]; then git submodule update --init --depth=1 --jobs=8 "$path"; fi'
```

Comment:
why do you keep changing this? the one i mentioned was correct

Reply:
In fact you can even add  !/SDL/

Reply:
it was failing

Reply:
I think you're confusing a git error with a string error.
The awk should be working fine.
It lookes like the local repo isn't synced properly.

Reply:
so ci issue?

Reply:
or an issue with the fact linux wants a quote around this

Reply:
revert it to `$(awk '/path/ && !/FAudio/ && !/llvm/ && !/SDL/ && !/SPIRV/ { print $3 }' .gitmodules)`
you may also add a line:
`echo $(awk '/path/ && !/FAudio/ && !/llvm/ && !/SDL/ && !/SPIRV/ { print $3 }' .gitmodules)`
above to see what it resolves to.

Reply:
if i revert it back to ` $(awk '/path/ && !/FAudio/ && !/llvm/ && !/SDL/ && !/SPIRV/ { print $3 }' .gitmodules)`
linux complains about there needing to be a double quote around the $(awk ...)

Reply:
In .ci/build-mac.sh line 54:
git submodule -q update --init --depth=1 --jobs=8 $(awk '/path/ && !/FAudio/ && !/llvm/ && !/SPIRV/ { print $3 }' .gitmodules)
                                                  ^-- SC2046 (warning): Quote this to prevent word splitting.